syncano/models/manager.py

```diff
@@ -828,6 +828,26 @@ def _get_endpoint_properties(self):
         return self.model._meta.resolve_endpoint(self.endpoint, defaults), defaults
 
 
+class RuntimeChoices(object):
```

Comment:
@forgems I prefer not to make queries from fields :( It's tricky business and LIB is not well desinged for that.